mantidimaging/gui/windows/spectrum_viewer/view.py

```diff
@@ -371,3 +434,28 @@ def set_binning_visibility(self) -> None:
         self.bin_size_spinBox.setHidden(hide_binning)
         self.bin_step_label.setHidden(hide_binning)
         self.bin_step_spinBox.setHidden(hide_binning)
+
+    def set_roi_properties(self) -> None:
+        current_roi = self.presenter.model.get_roi(self.current_roi)
+        self.roiPropertiesGroupBox.setTitle(f"Roi Properties: {self.current_roi}")
+        roi_iter_order = ["Left", "Top", "Right", "Bottom"]
+        for row, pos in enumerate(current_roi):
```

Comment:
If I delete all ROI's within ROI mode, We should clear the values from the spin boxes:
![image](https://github.com/mantidproject/mantidimaging/assets/32413847/3da5b05b-1ae0-469e-8230-44423acefca4)